src/main/java/us/eunoians/mcrpg/command/parser/SkillParser.java

```diff
@@ -73,16 +71,6 @@ public SkillParseException(final @NotNull String input, final @NotNull CommandCo
                     Caption.of("argument.parse.failure.skill"),
                     CaptionVariable.of("input", input)
             );
-            this.input = input;
-        }
-
-        /**
-         * Returns the supplied input.
-         *
-         * @return string value
-         */
-        public String input() {
```

Comment:
Wait why remove this? It's unused but is just a utility method in case others need it for this in the future